homeassistant/components/sensor/dyson.py

```diff
@@ -183,3 +204,101 @@ def state(self):
         if self._device.environmental_state:
             return self._device.environmental_state.volatil_organic_compounds
         return None
+
+
+class DysonParticulateMatter25Sensor(DysonSensor):
```

Comment:
These sensors should instead be part of the air pollutants component which is being renamed to air quality component.

So make a new platform under that component.